feature-group-definitions/file-and-directory-entries.yml

```diff
@@ -0,0 +1,38 @@
+name: File and directory entries
+description: File and directory entries represent a virtual file system for working with files and directories dragged and dropped into the browser.
+spec: https://wicg.github.io/entries-api/
+compat_features:
```

Comment:
Can you add api.HTMLInputElement.webkitEntries and api.DataTransferItem.webkitGetAsEntry as long as that doesn't change support status? Those are the entry points to get a `FileSystemEntry` object.

Reply:
Done with 1598243.